Blazor.PWA.MSBuild.Tasks/build/BlazorPWA.MSBuild.ServiceWorker.targets

```diff
@@ -105,7 +105,7 @@
     <WriteLinesToFile Condition="'$(BlazorProjectType)' == 'CSB'"
                       File="$(WWWRoot)$(ServiceWorkerFileName)"
                       Overwrite="false"
-                      Lines="const requiredFiles = [;@(DistFiles -> '&quot;/%(RecursiveDir)%(FileName)%(Extension)&quot;'->Replace('\','/'),',%0D%0A');]%3B;" />
```

Comment:
This looks odd - why remove RecursiveDir? it is needed to get the full path to the file. Mistake?

Reply:
Looks like I read your proposed change in gitter wrong!

Reply:
fixed